helm/insights-api-db-svc/values/values-test.yaml

```diff
@@ -29,6 +29,7 @@ resources:
     memory: 16Gi
 
 taskTimeout: 48h
-maxParallelTasks: 15
+# decreasing maxParallelTasks helps to reduce IO/MEM load in insights database, but slowers new indicator processing
+maxParallelTasks: 11
```

Comment:
Other tweaks suggest 2/3 reduction, so this one logically should've become 10 (2/3*15) not 11 -- but its a pure nitpicking. The range is the almost the same so supposed quantitative impact would still be easy to spot.

Its just a remark, not a blocker.